unit/analyses/dependence_graph.cpp

```diff
@@ -8,15 +8,15 @@ Author: Chris Smowton, chris.smowton@diffblue.com
 
 #include <iostream>
 
-#include <testing-utils/catch.hpp>
 #include <analyses/dependence_graph.h>
-#include <util/symbol_table.h>
-#include <util/std_code.h>
-#include <util/c_types.h>
-#include <util/arith_tools.h>
 #include <goto-programs/goto_convert_functions.h>
+#include <java_bytecode/java_bytecode_language_info.h>
 #include <langapi/mode.h>
-#include <java_bytecode/java_bytecode_language.h>
+#include <testing-utils/catch.hpp>
+#include <util/arith_tools.h>
+#include <util/c_types.h>
+#include <util/std_code.h>
+#include <util/symbol_table.h>
```

Comment:
Nit pick: I'm not sure this is a value-adding change? We usually do grouping by directories.